CmrUI/Screens/SettingsTab/SettingsVC.swift

```diff
@@ -43,8 +43,7 @@ class SettingsVC: UIViewController {
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
         
-        GradientHelper.addGradient(onTopOf: gradientView,
-                                   colors: [constants.darkGradientColor,
+        gradientView.addGradient(colors: [constants.darkGradientColor,
```

Comment:
@vmsavka Why call this in viewDidAppear? Can't you do it in viewDidLoad and then count on viewDidLayoutSubviews to update the bounds?